tests/pipeline/snapshots/test_generate/test_kafka_connect_sink_weave_from_topics/pipeline.yaml

```diff
@@ -1,5 +1,5 @@
 - _cleaner:
-    name: streams-app
+    name: streams-app-v2
```

Comment:
not necessary to rename them

Reply:
The problem is that the [defaults.yaml](https://github.com/bakdata/kpops/blob/36c4c88ab7a00397a8e36656a117305e373a88f3/tests/pipeline/resources/defaults.yaml#L2-L2) sets the name based on the component type
```
pipeline-component:
  name: ${component.type}
```

Reply:
then it's alright